test/journeys/specs/smoke/widget-recents/index.js

```diff
@@ -105,7 +103,24 @@ describe('Smoke Tests - Recents Widget', () => {
     });
   });
 
+  describe('No Spaces Message', () => {
+    it('has no spaces title', () => {
+      assert.isTrue(browserLocal.element(elements.noSpacesTitle).isVisible(), 'does not have no spaces title');
+      assert.equal(browserLocal.element(elements.noSpacesTitle).getText(), 'No spaces yet');
+    });
+
+    it('has no spaces message', () => {
+      assert.isTrue(browserLocal.element(elements.noSpacesMessage).isVisible(), 'does not have no spaces message');
+      assert.equal(browserLocal.element(elements.noSpacesMessage).getText(), 'Create a space using the plus button next to the search bar above.');
+    });
+  });
+
   describe('Group Space', () => {
+    it('creates spaces', () => {
```

Comment:
Want to try switching back to `before` blocks or is it a hopeless cause with wdio v4 + multiremote?

Reply:
resolved via teams to leave for now